app/models/spotlight/blacklight_configuration.rb

```diff
@@ -170,7 +171,13 @@ def default_blacklight_config
     end
 
     protected
-    
+
+    def spotlight_image_version_fields
+      @@spotlight_image_derivatives.map do |version|
```

Comment:
Does this need to call `@@spotlight_image_derivatives`, or could it use the accessor method?